config/environments/production.rb

```diff
@@ -67,7 +67,7 @@
   # config.force_ssl = true
 
   # Set to :debug to see everything in the log.
-  config.log_level = :info
+  config.log_level = :debug
```

Comment:
Do we want to change the log level for prod?

Reply:
Thanks for spotting have reverted this back to `:info`